### ISSUE TITLE
Training loss increasing instead of decreasing

### ISSUE THREAD

**Issue by MatteoCappe** [eric-yyjau/pytorch-superpoint#123]
I'm training on a custom dataset made of grey-scale images coming from the DSEC dataset. I am having trouble fully understanding how the training work, has this problem ever occured to other people? Or am I the only one?

For reference, I created a config.yaml file in which I defined my dataset and I disabled all data augmentation as I thought that it was causing problems.
I uploaded also an image depicting the loss per epoch

![Image](https://github.com/user-attachments/assets/de800e21-9979-4233-acad-b2f450cbe81c)
